plugins/cpbase64.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  # 'name' must match the filename of the manifest. The name defines how
```

Comment:
Can you please remove the example comments from the manifest?

Reply:
Done